packages/material-ui/src/locale/index.ts

```diff
@@ -1680,60 +1680,60 @@ export const skSK: Localization = {
 
 export const svSE: Localization = {
   props: {
-    // MuiBreadcrumbs: {
-    //   expandText: 'Show path',
-    // },
+    MuiBreadcrumbs: {
+      expandText: 'Visa sökväg',
+    },
     MuiTablePagination: {
-      // getItemAriaLabel: (type) => {
-      //   if (type === 'first') {
-      //     return 'Go to first page';
-      //   }
-      //   if (type === 'last') {
-      //     return 'Go to last page';
-      //   }
-      //   if (type === 'next') {
-      //     return 'Go to next page';
-      //   }
-      //   // if (type === 'previous') {
-      //   return 'Go to previous page';
-      // },
+      getItemAriaLabel: (type) => {
+        if (type === 'first') {
+          return 'Gå till första sidan';
+        }
+        if (type === 'last') {
+          return 'Gå till sista sidan';
+        }
+        if (type === 'next') {
+          return 'Gå till nästa sida';
+        }
+        // if (type === 'previous') {
+        return 'Gå till föregående sida';
+      },
       labelRowsPerPage: 'Rader per sida:',
       labelDisplayedRows: ({ from, to, count }) =>
-        `${from}-${to} av ${count !== -1 ? count : `more than ${to}`}`,
+        `${from}-${to} av ${count !== -1 ? count : `mer än ${to}`}`,
```

Comment:
I think you mean `fler än`.

As I interpret it, this is related to the amount of pages. The amount of pages is an exact quantity, and therefore I believe it should be `fler än` rather than `mer än`. I could be wrong though.

[Here is my source.](http://www.isof.se/sprak/sprakradgivning/frageladan.html?url=-27634753%2Fcgi-bin%2Fsrfl%2Fvisasvar.py%3Fsok%3Dmer%2Beller%2Bmera%26svar%3D26638)

Reply:
It's the number of rows, but I think you're right. I'll change it.